retroshare-gui/src/gui/gxs/GxsGroupShareKey.cpp

```diff
@@ -93,9 +93,9 @@ void GroupShareKey::setTyp()
         if (!rsPosted)
             return;
         
-        ui->headerFrame->setHeaderImage(QPixmap(":/images/posted_64.png"));    
-        ui->headerFrame->setHeaderText(tr("Share topic admin permissions"));
-        ui->sharekeyinfo_label->setText(tr("You can allow your friends to edit the topic. Select them in the list below. Note: it is not possible to revoke Posted admin permissions."));
+        ui->headerFrame->setHeaderImage(QPixmap(":/icons/png/posted.png"));    

```

Comment:
There is no admin permission for boards (there is on channels, but not in Boards). Could you hide that menu entry?